discord_connector.py

```diff
@@ -277,6 +279,78 @@ def _handle_ban_user(self, param):
 
         return status
 
+    def _handle_fetch_message_history(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        channel_id = param['channel_id']
+
+        status_after, fetching_start_date = self.parse_date(param.get('fetching_start_date', None))
+        status_before, fetching_end_date = self.parse_date(param.get('fetching_end_date', None))
+
+        limit = param.get('limit', None)
+        if limit == 0:
+            limit = None
+        oldest_first = param.get('oldest_first', False)
+
+        self.save_progress(
+            "status_after {}: {} | status_before {}: {}".format(status_after, fetching_start_date, status_before,
+                                                                fetching_end_date))
+        if not (status_after and status_before):
+            return action_result.set_status(phantom.APP_ERROR,
+                                            "action result: fetching messages from {} channel ended with failure, unable to format date"
+                                            .format(channel_id))
+
+        status, messages = self.fetch_message_history(channel_id, action_result, fetching_start_date, fetching_end_date,
+                                                      oldest_first, limit)
+        if not status:
+            return action_result.set_status(phantom.APP_ERROR,
+                                            "action result: fetching messages from {} channel ended with failure".format(
+                                                channel_id))
+
+        for message in messages:
+            action_result.add_data({
+                "message id": message.id,
+                "author id": message.author.id,
+                "created at": str(message.created_at),
+                "embeds_attachments": True if (message.attachments or message.embeds) else False,
+                "content": message.content
+            })
+
+        summary = action_result.update_summary({})
+        summary['action result: '] = "action result: fetching messages from {} channel ended with success".format(
```

Comment:
`f-string`

Reply:
It seams that .format() is standardized way of handling strings, however it may be because it is older than f-string itself. I can change those to f-string if there is a need for that.

Reply:
In other places you have `f-strings` already. Let's stick to the modern solutions :) `.format` can be useful in some cases, but this is not one of such.